Dockerfile

```diff
@@ -30,6 +30,7 @@ COPY semantic-json semantic-json/
 COPY semantic-python semantic-python/
 COPY semantic-source semantic-source/
 COPY semantic-tags semantic-tags/
+COPY semantic-analysis semantic-analysis/
```

Comment:
1. Can we alphabetize this please? 🙏
2. Is it reasonable to copy `semantic*` or something like that?

If copying all of these isn’t reasonable, then NB @aymannadeem, you’ll have to edit this file in #243 to avoid the same issues cropping up after it’s merged.

Reply:
The only reason for doing the `--only-dependencies` build is to speed things up for those of us that release semantic to the GPR. Otherwise all this could be removed in favor of https://github.com/github/semantic/pull/342/files#diff-3254677a7917c6c01f55212f86c57fbfR39-R40